tests/tasks/test_notebook.py

```diff
@@ -150,25 +150,39 @@ def test_notebook_conversion_stores_as_unicode(tmp_directory, monkeypatch):
 
 
 @pytest.mark.parametrize(
-    "name, out_dir",
+    "name, out_dir, engine",
     [
-        ["sample.py", "."],
-        ["sample.R", "."],
-        ["sample.ipynb", "."],
+        ["sample.py", ".", "papermill"],
+        ["sample.R", ".", "papermill"],
+        ["sample.ipynb", ".", "papermill"],
         # check still works even if the folder does not exit yet
-        ["sample.ipynb", "missing_folder"],
+        ["sample.ipynb", "missing_folder", "papermill"],
+        # For Ploomber Engine
+        ["sample.py", ".", "ploomber_engine"],
+        pytest.param(
+            "sample.R",
+            ".",
+            True,
+            marks=pytest.mark.xfail(reason="Ploomber Engine doesn't support R"),
+        ),
```

Comment:
let's remove this test and instead let's add one that verifies that an error is raised when attempting to run an R notebook with ploomber-engine

you can access the language by inspecting the source property (https://github.com/ploomber/ploomber/blob/b7d48028690463c2feda1f5aacf88e39e5029984/src/ploomber/sources/notebooksource.py#L541)